libraries/package-deps-hash/src/getPackageDeps.ts

```diff
@@ -1,86 +1,148 @@
 import * as child_process from 'child_process';
 import { IPackageDeps } from './IPackageDeps';
 
-const PROCESS_OUTPUT_DELIMITER: string = '///~X~X~X~X~X~X~///';
+export type GitStatusChangeType = 'D' | 'M' | 'A';
 
-/** @public */
+/**
+ * Parses the output of the "git ls-tree" command
+ */
+export function parseGitLsTree(output: string): Map<string, string> {
+  const changes: Map<string, string> = new Map<string, string>();
+
+  if (output) {
+    // Note: The output of git ls-tree uses \n newlines regardless of OS.
+    output.split('\n').forEach(line => {
+
+      // A line is expected to look like:
+      // 100644 blob 3451bccdc831cb43d7a70ed8e628dcf9c7f888c8    src/typings/tsd.d.ts
+
+      if (line) {
+        // Take everything after the "100644 blob", which is just the hash and filename
+        const [hash, filename]: string[] = line.substr(line.indexOf('blob ') + 5).split('\t');
```

Comment:
That's in a different PR, which was the one I was trying to merge in the first place.